README.md

```diff
@@ -233,6 +233,14 @@ It will write `tokenRulesAddress` to your config file.
 * Replace `eip20Token` with eip20Token address.
 * Replace `organization` with an organization contract address. 
 
+## Tests
+
+To run the tests:
+
+1. Copy `./test/config_init.json.dist` to `./test/config_init.json`
```

Comment:
I would propose to automate it as much as possible.
We could insist that for running tests the values in config are there and valid.